docs/developer/permissions.mdx

```diff
@@ -0,0 +1,156 @@
+
+Examples of groups:
+- Translators - `MANAGE_TRANSLATIONS` permission.
+- Customer support - `MANAGE_ORDERS` and `MANAGE_USERS` permissions.
+
+### Creating and removing groups
+To create a new group, use [permissionGroupCreate](developer/api-reference/mutations/permission-group-create.mdx).
+
+Request:
+```graphql
+mutation {
+  permissionGroupCreate(
+    input: {
+      addPermissions: [MANAGE_GIFT_CARD, MANAGE_DISCOUNTS]
+      addUsers: []
+      name: "Sale managers"
+    }
+  ) {
+    errors {
+      message
+    }
+    group {
+      id
+      name
+      permissions {
+        name
+      }
+    }
+  }
+}
+```
+
+Successful response:
+```json
+{
+  "data": {
+    "permissionGroupCreate": {
+      "errors": [],
+      "group": {
+        "id": "R3JvdXA6NDM=",
+        "name": "Sale managers",
+        "permissions": [
+          {
+            "name": "Manage sales and vouchers.",
+          },
+          {
+            "name": "Manage gift cards.",
+          }
+        ]
+      },
+    }
+  }
+}
+```
+
+To remove group, use [permissionGroupDelete](developer/api-reference/mutations/permission-group-delete.mdx) mutation:
+
+```graphql
+mutation {
+  permissionGroupDelete(
+    id: "R3JvdXA6NDM="
+  ) {
+    errors {
+      message
+    }
+}
+```
+
+### Modifying group and managing its members
+
+[permissionGroupUpdate](developer/api-reference/mutations/permission-group-update.mdx) takes a list of user IDs you would like to add or remove from the group. Having a same user in both of the lists will result in an error.
+
+Example request:
+```graphql
+mutation {
+  permissionGroupUpdate(
+    id: "R3JvdXA6NDM="
+    input: {
+      name: "Sale managers"
+      addPermissions: []
+      removePermissions: []
+      addUsers: ["VXNlcjozMg=="]
+      removeUsers: []
+    }
+  ) {
+    errors {
+      message
+    }
+  }
+}
+```
+
+## App permissions
+
+In contrast to users, app permissions are defined per-app basis. Access can be assigned during app installation and modified later using [appUpdate](developer/api-reference/mutations/app-update.mdx) mutation.
+
+
+## JWT token and permissions
+
+JWT tokens have a list of assigned permissions. By decoding payload using HS256 algorithm you will get:
+
+```json
+{
+  "iat": 1624013260,
+  "iss": "example.com",
+  "token": "AixxXXXxzF",
+  "email": "john@example.com",
+  "type": "access",
+  "user_id": "VXNlcjozMg==",
+  "is_staff": true,
+  "exp": 1624049260,
+  "oauth_access_key": "",
+  "permissions": [
+    "MANAGE_TRANSLATIONS",
+    "MANAGE_PRODUCTS",
+    "MANAGE_PRODUCT_TYPES_AND_ATTRIBUTES"
+  ]
+}
+```
+
+To check token online and learn more about JWT visit [https://jwt.io](https://jwt.io).
+
+:::note
+Since Saleor reads permissions from the JWT, generating new token is nessesary when user permissions change.
```

Comment:
```suggestion
Since Saleor reads permissions from the JWT token, generating a new token is necessary when the user changes permissions.
```

Reply:
JWT stands for Json Web Token. No need to add token gain